neurodamus/modification_manager.py

```diff
@@ -96,7 +96,8 @@ def __init__(self, target, mod_info: dict, cell_manager):
                     continue
                 sec = sc.sec
                 if all(hasattr(sec, x) for x in config_attrs):  # if has all attributes
-                    exec(config, {"__builtins__": None}, {"sec": sec})  # unsafe but sanitized
+                    # unsafe but sanitized
```

Comment:
I disagree that this is sanitized; so I think this comment is misleading.  We have to assume, IMO, that anyone can execute code in python by this method.